templates/index.html

```diff
@@ -59,6 +59,14 @@ <h1 class="post-title">
                         </h1>
 
                         <span class="post-date">{{ page.date | date(format="%Y-%m-%d") }}</span>
+
+                        <span class="post-summary">
+                          {% if page.description %}
```

Comment:
i would also put `page.summary` as the first condition as `description` is something used more for SEO/metadata typically